packages/mobile/src/components/core/Text.tsx

```diff
@@ -85,5 +86,14 @@ export const Text = (props: TextProps) => {
     ]
   )
 
-  return <RNText style={[styles.root, customStyles, style]} {...other} />
+  const children =
+    typeof childrenProp === 'string'
+      ? childrenProp.replace('\n', ' ')
```

Comment:
Neat! How should we be doing line breaks in our own messages? Do you always do two separate messages spaced using flexbox or is there another way?

Reply:
its a good question, actually we should probably have a prop here to disable this, since i see some areas of the mobile code where we do explicit line breaks. Ill fix those entries. Do you have an idea on if it should be default on vs off?

Reply:
I think default on is good to prevent the bug reoccurring. As long as it's not too much work to find the existing newline usages, I would go with that.

Reply:
Better to require desired newline cases to be explicit about it so we never do it by accident.